README.md

```diff
@@ -12,8 +12,9 @@ Reverting to stock firmware is also possible [see here](https://github.com/mongo
 Summary of supported devices and features
 
 ||[1]|[1PM]|[1L]|[Plug]|[PlugS]|2|[2.5]|[i3]|[RGBW2]|
-|-|-|-|-|----|-----|-|---|--|-----|
+|-|-|-|-|-|-|-|-|-|-|
 |Switch & Co.|✓|✓|✓|✓|✓|✓|✓|✗|✗|
+|Temperature measurement|✓|✓|✗|✗|✗|✗|✗|✗|✗|
```

Comment:
this may be confusing because there's an internal temperature sensor too, which is supported on more models.
let's call this Sensor AddOn and link to the page - https://shop.shelly.cloud/temperature-sensor-addon-for-shelly-1-1pm-wifi-smart-home-automation#312

Reply:
I get your point, but my opinion is the result and not the technic, if we add the Shelly H&T I would add it to that row as well.

What do you think about "Environment temperature"?

Reply:
in practice though, you do need the sensor add-on to attach sensors, don't you?
i think it's fair to call it "Sensor AddOn", and visiting the link will clarify what's it for.

Reply:
How would you do it when the H&T comes?

What about "Room temperature"?

Reply:
we'll think about H&T when it comes.
but simply saying room temperature measurement on 1 and 1PM is supported without mentioning the add-on is misleading.

i think what we can do is call the row "Temperature Measurement" and add little asterisk next to the checkmark, liek so:

```
|Temperature measurement|✓(*)|✓(*)|✗|✗|✗|✗|✗|✗|✗|

*: With [Sensor AddOn](https://shop.shelly.cloud/temperature-sensor-addon-for-shelly-1-1pm-wifi-smart-home-automation#312)
```

bonus points if you can bend the markup to make superscript "1" instead of (*).

also, we should remove "temperature addon" from the list of unsupported stuff below.